chips/sam4l/src/gpio.rs

```diff
@@ -79,6 +80,8 @@ pub enum PeripheralFunction {
 const BASE_ADDRESS: usize = 0x400E1000;
 const SIZE: usize = 0x200;
 
+pub static INTERRUPT_COUNT: AtomicUsize = AtomicUsize::new(0);
```

Comment:
I see. So we're actually counting the number of interrupts that are enabled across each pin of a port and across all ports in the GPIO peripheral.